app/instrumentation/aiobotocore/__init__.py

```diff
@@ -0,0 +1,297 @@
+# Based on botocore instrumentation:
```

Comment:
Can we try to patch it instead of copy-pasting code? Contributing upstream might take some time.